scalajslib/src/mill/scalajslib/ScalaJSModule.scala

```diff
@@ -75,6 +75,11 @@ trait ScalaJSModule extends scalalib.ScalaModule { outer =>
     val commonDeps = Seq(
       ivy"org.scala-js::scalajs-sbt-test-adapter:${scalaJSVersion()}"
     )
+    val maybeImportMap = scalaJSVersion() match {
+      case s"1.$n.$_" if n.toIntOption.exists(_ < 16) => Seq[Dep]()
+      case _ => Seq(ivy"com.armanbilge::scalajs-importmap:0.1.1")
```

Comment:
Two things:
1. This needs to be added only when there are defined mappings, so users who don't use the feature don't need to download the dependency.
2. The version needs to come from `BuildInfo` so it is updated only in `build.sc`